content/docs/overview/component/filter-gateway.md

```diff
@@ -26,6 +26,7 @@ Ingress filtering is the pre-process before the request is processed by the Vald
 This is useful, e.g., when you want to vectorize object data (when you want to handle object data directly as the request query) or when you want to pad the vector, etc.
 
 Here are the steps of pre-processing with the ingress filter:
+
```

Comment:
seems the image below is dead.
https://github.com/vdaas/web/pull/116/files#diff-9f4fffa70fce0b833f4ccbade8f2f5bcc5c2715d57e672a8e4af6dfa768f031aR34-R36

Reply:
???

Reply:
sorry my mistake